rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/dto/TaskStateData.java

```diff
@@ -125,10 +129,12 @@ public void setParallelEnvironment(ParallelEnvironmentData parallelEnvironment)
         this.parallelEnvironment = parallelEnvironment;
     }
 
-    public Map<String, String> getGenericInformations() { return genericInformations; }
+    public Map<String, String> getGenericInformation() {
+        return genericInformation;
+    }
 
-    public void setGenericInformations(Map<String, String> genericInformations) {
-        this.genericInformations = genericInformations;
+    public void setGenericInformation(Map<String, String> genericInformation) {
```

Comment:
Changing the method name prevents backward compatibility.

You could add the old method with a @Deprecated annotation to prevent that.